client/src/components/Module.jsx

```diff
@@ -62,17 +88,16 @@ function Module(props) {
 Module.propTypes = {
   title: PropTypes.string.isRequired,
   body: PropTypes.string.isRequired,
-  attachments: PropTypes.string.isRequired,
   child: PropTypes.arrayOf(PropTypes.shape({
     id: PropTypes.string.isRequired,
     title: PropTypes.string.isRequired,
     role: PropTypes.arrayOf(string).isRequired,
   })).isRequired,
-  link: PropTypes.string,
+  links: PropTypes.arrayOf(string),
 };
 
 Module.defaultProps = {
-  link: '',
+  links: [''],
```

Comment:
I think this should just be an empty array, since u check if links has size 0 on line 19